README.md

```diff
@@ -82,6 +82,17 @@ should be called to update the UI.
 This in turn will invoke the `build` method 
 which re-renders the widget.
 
+## Flutter application test
+
+As in all languages or platforms the secret to a successful application is to test it.
+Several tests should be done like the functionality test, the user experience test, which can help us correct any bug.</br>
```

Comment:
```suggestion
Several tests should created included functional tests and UX tests 
which can help us discover and fix any bugs before users see them!</br>
```
slight improvement to flow and meaning of the sentence.
also, splitting the sentence into semantic line feeds. 
Please read: https://sembr.org

Reply:
Thanks for the help, writing English is still a little complicated.